Core-Usage.md

```diff
@@ -0,0 +1,147 @@
+#Stanwood Core
+
+##Project Organisations
+Each Xcode project should be set with the following group structure in that order. This is so all our projects are consistent. [Project Organisation](https://www.figma.com/proto/AO3XMoeps1Yjj7aia5HpmA/Untitled?node-id=31%3A107&scaling=min-zoom)
+
+###Utilities
+Always check that StanwoodCore does not provide this for free. Extensions, Helpers
+###Services
+Tracking, DFP, Networking(dataProvider), etc', 
+###Model
+Should contain `ModelCollection`, and `Model` types
+###Views
+Should contain any global custom views used
+###Modules
+Each module should be set as a sub-group and contains ModuleViewController, ModuleWireframe, ModulePresenter, ModuleActionable ,ModuleParamaterable, ModuleViewable, Module.storyboard, ModuleDataSource ,ModuleDelegate, ModuleCells 
+###Stanwood
```

Comment:
```suggestion
### Stanwood
```

Reply:
I think this is deprecated. We changed it to `Core` if I am not mistaken

Reply:
Ah, I forgot about that. How you want to structure that? Just a link the the repo and a brief overview ?

Reply:
> Core